recipes/gdal2/meta.yaml

```diff
@@ -0,0 +1,46 @@
+package:
+  name: gdal
+  version: 2.1.0.trunk
+  
+source:
+  svn_url: https://svn.osgeo.org/gdal/trunk/gdal
```

Comment:
We should used only released versions. The latest one 2.0.2 is here: http://trac.osgeo.org/gdal/wiki/DownloadSource

(Any special reason for using the latest source? If so can you use the new GitHub repository?)